### PR TITLE
fix: text to be more concise #160

### PR DESCRIPTION
**Pull request by EthenThinkful**
## Description

Updated text in the "The Official Kickoff" section of 100-days-of-oss.md to be more comprehensible for newcomers. This ensures anyone brand new to open source can easily understand the timeline of 100DaysOfOSS. 

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [✔] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Mobile & Desktop Screenshots/Recordings

Before text change: 
isssue1

After text change:
fix-issue-1

## Added tests?

- [ ] 👍 yes
- [✔] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [✔] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "workbench.colorCustomizations": {
+        "activityBar.background": "#083606",
+        "titleBar.activeBackground": "#0B4B09",
+        "titleBar.activeForeground": "#F2FDF1"
+    }
+}
```
Comment (CBID2) [inline .vscode/settings.json:7]: I highly recommend deleting this file @EthenThinkful

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @EthenThinkful. Congrats on starting up your open source journey again. I just implemented a suggestion on how to improve it. Other than that, you're great.

File: .vscode/settings.json (hunk 1 above)
Comment (EthenThinkful) [reply .vscode/settings.json]: deleted!

**Review by CBID2 — APPROVED** [review]
Looks great!

**Comment by BekahHW** [comment]
@EthenThinkful if there's an issue related to the PR, make sure to include that in your description.

For instance:

closes #160

**Pull request merged**